doc/man3/OPENSSL_ia32cap.pod

```diff
@@ -112,9 +112,9 @@ a.k.a. AVX512IFMA extension;
 
 =back
 
-To control this extended capability word use ':' as delimiter when
+To control this extended capability word use ":" as delimiter when
 setting up OPENSSL_ia32cap environment variable. For example assigning
-':~0x20' would disable AVX2 code paths, and ':0' - all post-AVX
+":~0x20" would disable AVX2 code paths, and ":0" - all post-AVX
```

Comment:
'~' runs into homedir expansion, though IIRC it is passed unchanged when no expansion is performed (as opposed to '!' which errors).